example-get-started/code/src/evaluate.py

```diff
@@ -55,14 +58,20 @@
         indent=4,
     )
 
-with open(roc_file, "w") as fd:
-    json.dump(
-        {
-            "roc": [
-                {"fpr": fp, "tpr": tp, "threshold": t}
-                for fp, tp, t in zip(fpr, tpr, roc_thresholds)
-            ]
-        },
-        fd,
-        indent=4,
-    )
+
+# ... confusion matrix plot
+predictions = [{
+                "actual": int(actual),
+                "predicted": 1 if predicted > 0.5 else 0
+               } for actual, predicted in zip(labels, predictions)]
+with open("evaluation/plots/predictions.json", "w") as f:
+    json.dump(predictions, f)
```

Comment:
```suggestion
live.log_plot("confusion_matrix", labels, predictions_by_class.argmax(-1))
```

Reply:
Would need to replace `predictions` with `confusion_matrix` elsewhere

Reply:
done, thanks, I will give it a try